src/main/docker/Dockerfile

```diff
@@ -5,5 +5,5 @@ VOLUME /tmp
 ADD portal-exec.jar /portal.jar
 ADD run.sh /bin/run.sh
 RUN chmod +x /bin/run.sh
-EXPOSE 8080 5434 8983
-CMD ["/bin/run.sh"]
\ No newline at end of file
+EXPOSE 8080:8443 5434 8983
```

Comment:
According to Docker's documentation, EXPOSE command has the following syntax:
```EXPOSE <port> [<port>/<protocol>...]```
What does 8080:8443 mean?

Reply:
This part is still to be fixed

Reply:
Fixed